io/module_wrt_grid_comp.F90

```diff
@@ -867,6 +891,17 @@ subroutine wrt_initialize(wrt_comp, imp_state_write, exp_state_write, clock, rc)
 
         if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
 
+! add iau here
```

Comment:
more informative comment please

Reply:
added comments "! update the time:units when idate on write grid component is changed"